docs/HowTo/Configure/Consensus-Protocols/IBFT.md

```diff
@@ -181,7 +180,22 @@ To tune the block timeout for your network deployment:
     View [`TRACE` logs](../../../Reference/API-Methods.md#admin_changeloglevel) to see round change
     log messages.
 
-#### Configure block time on an existing network deployment
+Use a [transition](#transitions) to update the `blockperiodseconds` in an existing network.
+
+{!global/Config-Options.md!}
+
+## Transitions
```

Comment:
I like that Transitions have their own section now, that'll be helpful for linking to it which I seem to be doing more often these days :)